backend/dgraph/src/configuration/delete_configuration_item.rs

```diff
@@ -0,0 +1,124 @@
+use gql_client::GraphQLError;
+use serde::{Deserialize, Serialize};
+
+use crate::DgraphClient;
+
+#[derive(Serialize, Debug, Clone)]
+struct DeleteVars {
+    code: String,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Clone)]
+pub struct UpsertResponseData {
+    pub data: UpsertResponse,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Clone)]
+pub struct UpsertResponse {
+    pub numUids: u32,
+}
+
+// Dgraph sometimes returns an error like this:
```

Comment:
This actually happens quite a bit of you're deleting lots of items.
I think we should maybe put in somekind of random but very small delay between retries like between 1-5ms?

Reply:
Arguably should also create some kind of helper function for this and/or see if we can set some kind of parameter to avoid these issues?